pkg/k8s/network_policy.go

```diff
@@ -19,7 +19,8 @@ import (
 
 	"github.com/cilium/cilium/pkg/annotation"
 	k8sConst "github.com/cilium/cilium/pkg/k8s/apis/cilium.io"
-	k8sUtils "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/utils"
+	k8sCiliumUtils "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/utils"
+	k8sUtils "github.com/cilium/cilium/pkg/k8s/utils"
```

Comment:
I see what you meant about the names. I think this is the right choice. I saw util and expected it to be called that so it made sense.